doc/installation.rst

```diff
@@ -4,15 +4,11 @@ Installation
 Requirements
 ------------
 
-Installing the **garnett** package requires python 2.7.x or 3.x, and **numpy**.
+Installing the **garnett** package requires python 2.7.x or 3.x, **numpy** and **rowan**.
```

Comment:
I recommend narrowing the support. Also capitalize Python and add a comma after numpy:

`Installing the **garnett** package requires Python 2.7 or 3.5+, **numpy**, and **rowan**.`

Reply:
Done

Reply:
@lyrivera I un-resolved this -- it doesn't look like anything changed?

Reply:
@bdice Should be now, not sure what happened